WSL/troubleshooting.md

```diff
@@ -267,6 +267,8 @@ When enabled, the following apply to proxy settings on your Linux distributions:
 - The Linux environment variable, `NO_PROXY`, is set to bypass any HTTP/S proxies found in the Windows configuration targets.
 - Every environment variable, except `WSL_PAC_URL`, is set to both lower case and upper case. For example: `HTTP_PROXY` and `http_proxy`.
 
+There is a known issue caused by ZScaler configurations, where ZScaler repeatedly enables and disables Windows proxy configurations, leading to WSL repeatedly showing the "An Http proxy change has been detected on the host" notification.
```

Comment:
Can we expect that our target audience here will know what ZScaler configurations are? Should we explain any further or provide a link?

Reply:
the issue that motivated this note was reported at https://github.com/microsoft/WSL/issues/11523

we know the issue is caused by ZScaler but we don't know how ZScaler was configured

should we rephrase this to "caused by ZScaler" ? and we can add the link to the WSL github issue above